src/oic/oauth2/__init__.py

```diff
@@ -68,7 +69,7 @@
     "AuthorizationRequest": "authorization_endpoint",
     "AccessTokenRequest": "token_endpoint",
     # ROPCAccessTokenRequest: "authorization_endpoint",
-    # CCAccessTokenRequest: "authorization_endpoint",
+    "CCAccessTokenRequest": "authorization_endpoint",
```

Comment:
This is the wrong endpoint, at least according to the RFC 6749 4.4.2
"(A)  The client authenticates with the authorization server and
        requests an access token from the **token endpoint**."